test/docker-test/omsfiles/custom.log

```diff
@@ -0,0 +1,20 @@
+2018-10-18 01:47:10 We need to rent a room for our party.
+2018-10-18 01:47:10 Yeah, I think it's a good environment for learning English.
+2018-10-18 01:47:10 Everyone was busy, so I went to the movie alone.
+2018-10-18 01:47:10 A purple pig and a green donkey flew a kite in the middle of the night and ended up sunburnt.
+2018-10-18 01:47:10 Yeah, I think it's a good environment for learning English.
+2018-10-18 01:47:10 I am never at home on Sundays.
+2018-10-18 01:47:10 There were white out conditions in the town; subsequently, the roads were impassable.
+2018-10-18 01:47:10 I hear that Nancy is very pretty.
+2018-10-18 01:47:10 There were white out conditions in the town; subsequently, the roads were impassable.
+2018-10-18 01:47:10 I am counting my calories, yet I really want dessert.
+2018-10-18 01:47:10 A purple pig and a green donkey flew a kite in the middle of the night and ended up sunburnt.
+2018-10-18 01:47:10 I am counting my calories, yet I really want dessert.
+2018-10-18 01:47:10 Cats are good pets, for they are clean and are not noisy.
+2018-10-18 01:47:10 Where do random thoughts come from?
+2018-10-18 01:47:10 I am never at home on Sundays.
+2018-10-18 01:47:10 The memory we used to share is no longer coherent.
+2018-10-18 01:47:10 She folded her handkerchief neatly.
+2018-10-18 01:47:10 The memory we used to share is no longer coherent.
+2018-10-18 01:47:10 She was too short to see over the fence.
+2018-10-18 01:47:10 Rock music approaches at high velocity.
```

Comment:
where did you find all of this 😂 @jamvar

Reply:
You have to creative sometimes. I got them using a random sentence generator from ruby libs I used previously.